requirements/dev.txt

```diff
@@ -1,9 +1,10 @@
-autopep8 ~= 2.0.1
-coverage ~= 7.1.0
-flake8 ~= 6.0
-ipython ~= 7.32
-isort ~= 5.5
-moto ~= 1.3
-pytest ~= 7.2.1
-python-dotenv ~=0.21
-safety ~= 1.10
+autopep8 ~= 2.0.4
+coverage ~= 7.3.2
+flake8 ~= 6.1.0
+ipython ~= 8.17.2
+isort ~= 5.12.0
+moto ~= 4.2.7
+pytest ~= 7.4.3
+python-dotenv ~=1.0.0
+safety ~= 2.3.5
+werkzeug ~= 3.0.1 # Dependency of moto, pinned to mitigate vulnerability.
```

Comment:
When running the installation on my local machine (in a Python 3.9 environment), `moto~=4.2.7` led to `werkzeug==3.0.1`, which mitigates the issue. However, Snyk still pulled in an older version of `werkzeug` when running the scan. I suspect it relates to Snyk running the scan under Python 3.7.

Reply:
This isn't great, but I'm following along with the slack chat to see if we can resolve it differently for the future.